packages/vite/src/node/importGlob.ts

```diff
@@ -161,7 +176,14 @@ function lexGlobPattern(code: string, pos: number): [string, number] {
         throw new Error('unknown import.meta.glob lexer state')
     }
   }
-  return [pattern, code.indexOf(`)`, i) + 1]
+  const endIndex = code.indexOf(`)`, i)
+  const options = code.substring(i + 1, endIndex)
+  const commaIndex = options.indexOf(`,`)
+  let assert = {}
+  if (commaIndex > -1) {
+    assert = JSON5.parse(options.substr(commaIndex + 1))
```

Comment:
```suggestion
    assert = JSON5.parse(options.substring(commaIndex + 1))
```

There was a recent PR that converted `substr` to `substring`. I think we can do the same here too.